metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
@justinleet the tool I mentioned (npm-licence-crawler) will crawl all transitive dependencies, and only popped the 3 clause variant on master + this PR, so looks like that's been handled properly. 

Perhaps we should go ahead with this one, and set a JIRA to do a full review of licensing and automation of the checks for npm related bits.